README.md

```diff
@@ -8,17 +8,28 @@
 [CS2103-F13-1] SpleetWaise
```

Comment:
```suggestion
# [CS2103-F13-1] SpleetWaise
```
Since we are using headings, I think we should make the application name a level 1 heading to make it stand out.